test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -12860,6 +12860,24 @@ def add_config(self, conf={}, hup=True):
 
         return self.apply_config(conf, hup)
 
+    def unset_mom_config(self, name, hup=True):
```

Comment:
@lisa-altair, @hirenvadalia  I think it is okay to have the name and behavior consistent with how it has been historically there for scheduler.
In future we would want all our configurations to move to qmgr and then these interfaces might not even be needed. For now, I think it is okay to have it as it is.